node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -562,6 +724,7 @@ mod tests {
 
     #[test]
     fn can_read_dns_servers_and_consuming_private_key_from_config_file() {
+        let _guard = EnvironmentGuard::new();
```

Comment:
These tests don't involve the environment directly; but they use `MultiConfig` code that checks the environment, and if there are other tests that do use the environment running in parallel, these tests can get confused. So I added `EnvironmentGuard`s.